lib/jekyll-webp/webpExec.rb

```diff
@@ -27,8 +27,8 @@ def self.run(quality, input_file, output_file)
         full_path = File.join(gem_root, bin_path, exe_name)
 
         # Construct the full program call
-        cmd = "\"#{full_path}\" -quiet -mt -q #{quality.to_s} \"#{input_file}\" -o \"#{output_file}\""
-        
+        cmd = "\"#{full_path}\" -quiet -mt -m 6 -pass 10 -q #{quality.to_s} \"#{input_file}\" -o \"#{output_file}\""
```

Comment:
Please expose these controls as configurable values to enable keeping existing behaviour.